dkms.in

```diff
@@ -1277,21 +1333,26 @@ do_install()
                 $"You may override by specifying --force."
         fi
         local m=${dest_module_name[$count]}
-        local installed_modules=$(find_module "$lib_tree" "$m")
+        local installed_modules
+        installed_modules=$(find_module "$lib_tree" "$m")
         local module_count=${#installed_modules[@]}
         echo $" - Original module"
-        local original_copy=$(compressed_or_uncompressed "$dkms_tree/$module/original_module/$kernelver/$arch" "$m")
+        local original_copy
+        original_copy=$(compressed_or_uncompressed "$dkms_tree/$module/original_module/$kernelver/$arch" "$m")
         if [[ -L $dkms_tree/$module/kernel-$kernelver-$arch &&
             -n "$original_copy" ]]; then
             echo $"   - An original module was already stored during a previous install"
         elif ! [[ -L $dkms_tree/$module/kernel-$kernelver-$arch ]]; then
-            local archive_pref1=$(compressed_or_uncompressed "$lib_tree/extra" "$m")
-            local archive_pref2=$(compressed_or_uncompressed "$lib_tree/updates" "$m")
-            local archive_pref3=$(compressed_or_uncompressed "$lib_tree${dest_module_location[$count]}" "$m")
+            local archive_pref1
+            archive_pref1=$(compressed_or_uncompressed "$lib_tree/extra" "$m")
+            local archive_pref2
+            archive_pref2=$(compressed_or_uncompressed "$lib_tree/updates" "$m")
+            local archive_pref3
+            archive_pref3=$(compressed_or_uncompressed "$lib_tree${dest_module_location[$count]}" "$m")
```

Comment:
Why is this style used rather than `local var=value`?

Reply:
I got "Declare and assign separately to avoid masking return values."  in shellcheck https://www.shellcheck.net/wiki/SC2155

Reply:
> Why is this style used rather than `local var=value`?

Maybe he want to workaround [the shellcheck bug](https://www.shellcheck.net/wiki/SC2178#bugs).